ui/webui/src/app.jsx

```diff
@@ -15,33 +15,88 @@
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
 import cockpit from "cockpit";
+
 import React, { useEffect, useState } from "react";
 
 import {
-    Page,
+    Page, Wizard,
 } from "@patternfly/react-core";
 
 import { InstallationLanguage } from "./InstallationLanguage.jsx";
-import { Summary } from "./Summary.jsx";
+import { InstallationDestination } from "./storage/InstallationDestination.jsx";
+import { ReviewConfiguration } from "./installation/ReviewConfiguration.jsx";
+import { InstallationProgress } from "./installation/InstallationProgress.jsx";
+import { InstallationComplete } from "./installation/InstallationComplete.jsx";
```

Comment:
Should this be one line?
`import { InstallationProgress, InstallationComplete } from "./installation/InstallationProgress.jsx";`

Reply:
Hmm, yeah, makes sense, lets do that. :)

Reply:
Actually, no I think ? Each jsx file has one of these top-level classes/functions, so you have to select the correct jsx file to import from.

Reply:
Since you keep these component in different files, you need also to import them in seperate lines as you do now. You could always have more than one components in one file, and in that has you can use this suggestion from vladimir.